src/app/app-modules/pharmacist/worklist/worklist.component.html

```
@@ -1,165 +1,45 @@
-              [zDisabled]="currentPage === totalPages"
-              aria-label="Next page"
-              (click)="nextPage()">
-              <ng-icon name="lucideChevronRight" size="16"></ng-icon>
-            </button>
-          </li>
-        </ul>
-      </div>
-    </z-card-content>
-  </z-card>
-</div>
+<app-beneficiary-worklist
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)